### ISSUE TITLE
Set a different power profile depending on if the device is in battery mode or AC mode

### ISSUE THREAD

**Issue by xaviergonz** [Valkirie/HandheldCompanion#835]
**Is your feature request related to a problem? Please describe.**
I try to use the default profile as much as possible, but I find myself changing the TDP all the time when the PC is in battery mode and I get it in AC mode and vice-versa.

**Describe the solution you'd like**
Being able to set a power profile depending on if the device is in battery mode or in AC mode.

**Describe alternatives you've considered**
Alternatively two default profiles (one for AC mode, another one for battery), but then I'd need to make sure they are in sync, since the only thing I want changed is the power profile.

**Comment by CasperH2O** [comment]
Agreed. There is actually an active PR at the moment which has a nice start for this: #833
